test/datadog.test.js

```diff
@@ -6,12 +6,6 @@ const sinon = require('sinon')
 
 const axios = require('axios')
 
-test('creates client', t => {
-  const client = new tested.Client({ apiKey: '1234567890' })
-  t.equals(client._apiKey, '1234567890')
```

Comment:
We are not supposed to access the `_apiKey` directly, right ?

if it is allowed, it is a regression.
It not, this case is now covered by the test "inserts sends com url and api key" where I check the apiKey is sended to the api.